docs/data-apps/index.md

```diff
@@ -53,3 +58,161 @@ Our apps provide useful help text throughout the apps, keep an eye out for the h
 
 ### Log Out
 If you wish to Log out of the application, you can do this on the `Account` page of any application. Note this also logs you out of Console.
+
+
+## One off-setup
+To setup the app and have it connect to your warehouse we require a user/role to run the app via, the below steps provide more information on this.
+
+Once you have purchased the Snowplow Digital Analytics package, or other package containing Data Apps, Snowplow’s Infrastructure Support team will contact you to review the setup details, and then deploy the data applications into your cloud environment. 
+
+### Warehouse permissions
+
+All data applications will require warehouse user credentials with read access to the tables powering the data applications. There are two options: 
+
+1. Reuse the Data Modeling User - in this case we will use the existing Data Modeling user you created during your on-boarding. Note that this user may need additional table access granted to it to use your own tables in any of the apps e.g. the [funnel builder](/docs/data-apps/funnel-builder/index.md). 
+2. [Recommended] Create a new Data Applications user  - you can also create a new warehouse user with more fine-grained permissions. To create this, you can run the following script and then pass us the details via the secure messaging in console.
+
+:::tip
+
+While possible to re-use the same user for each data app, it may be beneficial to have a user per app for easier management of access and for logging purposes. In this case, please alter the scripts below to name each user/role as required.
+
+:::
+
+
+<Tabs groupId="warehouse" queryString>
+<TabItem value="bigquery" label="BigQuery" default>
+
+
+Create and share a GCP Service Account with us that has the following roles attached to it:
+
+```hcl
+data_app_roles = [
+  "roles/bigquery.dataViewer",
+  "roles/bigquery.filteredDataViewer",
+  "roles/bigquery.jobUser",
+  "roles/bigquery.metadataViewer",
+  "roles/bigquery.readSessionUser"
+]
+```
+
+Once created share the following details back with us:
+
+1. Project ID
+2. BigQuery Dataset ID
+3. BigQuery Dataset Region (indicator of where the app should default into but is configurable in most apps once launched)
```

Comment:
Is there a comma missing in the brackets?

Reply:
Also in the cases below, the information relates to the schema, shouldn't it relate to the dataset ID here?